README.md

```diff
@@ -176,11 +176,11 @@ constructorio.tracker.trackSearchResultClick('dogs', {
 });
 ```
 
-| Parameter | Type |
-| --- | --- |
-| `name` | string |
-| `customer_id` | string |
-| `result_id` | string |
+| Parameter | Type | Required |
+| --- | --- | :---: |
+| `name` | string | ✔ |
+| `customer_id` | string | ✔ |
+| `result_id` | string | ✔ |
```

Comment:
`result_id` is not required,  more of a very nice to have

Reply:
Great catches here - updated the JSDoc as well.